game.css

```diff
@@ -1,16 +1,21 @@
+* {
+  font-family: 'Lateef', cursive;
+  font-size: 22px;
```

Comment:
use rem for a responsive design

Reply:
Mario :angry: 
I normally use but in this section, i can't 
the font will be so big and i need to write something like that "0.1234rem" :thinking: